### PR TITLE
Prim's Algorithm

### PR DESCRIPTION
**Pull request by vinothgokul**

### CODE REVIEW

**Comment by Bhavnaharitsa** [comment]
The title is improper, a new algorithm...what algorithm?
Please add proper titles and description whenever making a pull request and the commit message can be altered..
You can read the commit message guidelines online and the rules while contributing open source.

File: Prim's_Algorithm/Prim's_Algorithm.cpp [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+// A utility function to find the vertex with 
+// minimum key value, from the set of vertices 
+// not yet included in MST 
+int minKey(int key[], bool mstSet[]) 
+{ 
+	// Initialize min value 
+	int min = INT_MAX, min_index; 
+
+	for (int v = 0; v < V; v++)
```
Comment (Bhavnaharitsa) [inline Prim's_Algorithm/Prim's_Algorithm.cpp]: no single variables

File: README.md [hunk 2]
```diff
@@ -4,5 +4,6 @@ This is a library of various algorithms that are used in competitive programming
 ### sample algorithm
 [disjoint_set_union.cpp](Algorithms/disjoint_set_union/disjoint_set_union.cpp)
 
+[prim's_algorithm.cpp]
```
Comment (Bhavnaharitsa) [inline README.md:7]: please resolve conflicts

**Review by chanmol1999 — CHANGES_REQUESTED** [review]
Add inside algorithm folder

**Pull request closed without merging**